services/quotes.js

```diff
@@ -0,0 +1,55 @@
+const {
+  sellerName,
+  sellerOffer,
+  sellerReturnPolicy,
+  sellerShippingFee,
+} = require('./helper');
+
+/* eslint-disable no-plusplus */
+module.exports.createQuotes = (prices, sellers, limit) => {
+  const response = [];
+  const fetchSellerMeta = (num, tag, array) => {
+    if (tag === 'name') {
+      return sellerName(num, array);
+    }
+    if (tag === 'offer') {
+      return sellerOffer(num, array);
+    }
+    if (tag === 'returnPolicy') {
+      return sellerReturnPolicy(num, array);
+    }
+    if (tag === 'shippingFee') {
+      return sellerShippingFee(num, array);
+    }
+    return null;
+  };
```

Comment:
Does this function need to live inside the createQuotes function? Also, maybe a switch statement may look a little cleaner? You can also instead return it without the braces and move it to a single line, example...
`if (tag === 'name') return name(num, array)`;

Reply:
While I agree there are other ways of doing above, not sure if this is a gating issue. I follow above convention, and for sake of consistency would prefer persisting with above approach.

Reply:
As long as you are aware that the function gets created every time you invoke createQuotes.